.travis.yml

```diff
@@ -38,12 +38,12 @@ matrix:
            NUMPY_VERSION="1.10.4" SCIPY_VERSION="0.16.1" CYTHON_VERSION="0.25.2"
            PILLOW_VERSION="4.0.0" COVERAGE=true
       if: type != cron
-    # This environment tests the newest supported Anaconda release.
+    # This environment tests the newest supported Anaconda release (5.2.0).
     # It runs tests requiring pandas and PyAMG.
     # It also runs with the site joblib instead of the vendored copy of joblib.
-    - env: DISTRIB="conda" PYTHON_VERSION="3.6.2" INSTALL_MKL="true"
-           NUMPY_VERSION="1.14.2" SCIPY_VERSION="1.0.0" PANDAS_VERSION="0.20.3"
-           CYTHON_VERSION="0.26.1" PYAMG_VERSION="3.3.2" PILLOW_VERSION="4.3.0"
+    - env: DISTRIB="conda" PYTHON_VERSION="3.6.5" INSTALL_MKL="true"
+           NUMPY_VERSION="latest" SCIPY_VERSION="latest" PANDAS_VERSION="latest"
+           CYTHON_VERSION="latest" PYAMG_VERSION="3.3.2" PILLOW_VERSION="5.1.0"
```

Comment:
With the latest conda,
```
$ conda install numpy=latest
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - numpy=latest
```

I don't understand why you don't get errors in this PR.

Maybe, `numpy='*'` that should work, I think.

Reply:
@rth hmm `numpy="*"` doesn't seem to work. but the `latest` format only works because I catch it in `install.sh`.